spring-resteasy/pom.xml

```diff
@@ -204,6 +209,11 @@
                         <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I had a look at ResteasySpringIT.java and it's really just a set of http client tests so please get ridden of Arquillian (dependencies, arq-* profiles, arquillian.xml) and convert it to junit, working similar to BasicRuntimeTest, so we can add it to integration-testing.